CHANGELOG.md

```diff
@@ -14,6 +18,10 @@
 
 * **[Fix]** Fix typescript compiler errors related to const enum usage.
 
+#### Android
+
+* **[Feature]** Add the `Crash.trackError` method to send handled errors (with optional properties and attachments).
```

Comment:
This feature is only for native android sdk, not for RN sdk. We should remove it from changelog.